plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/codescan/sessionconfig/CodeScanSessionConfig.kt

```diff
@@ -132,14 +138,27 @@ class CodeScanSessionConfig(
         return bufferedReader.useLines { lines -> lines.count() }
     }
 
-    private fun zipFiles(files: List<Path>): File = createTemporaryZipFile {
+    private fun zipProject(files: List<Path>): File = createTemporaryZipFile {
         files.forEach { file ->
             val relativePath = file.relativeTo(projectRoot.toNioPath())
             LOG.debug { "Selected file for truncation: $file" }
             it.putNextEntry(relativePath.toString(), file)
         }
     }.toFile()
 
+    private fun zipFile(filePath: Path): File = createTemporaryZipFile {
+        val relativePath = filePath.relativeTo(projectRoot.toNioPath())
+        val virtualFile = VirtualFileManager.getInstance().findFileByNioPath(filePath)
```

Comment:
hey tai...we need to not use this findFileByNioPath. This would result in error when we are using this for remote development files.